src/main/java/com/mineboundteam/minebound/magic/UtilitySpells/TelekineticUtilitySpell.java

```diff
@@ -0,0 +1,116 @@
+package com.mineboundteam.minebound.magic.UtilitySpells;
+
+import com.mineboundteam.minebound.MineBound;
+import com.mineboundteam.minebound.config.IConfig;
+import com.mineboundteam.minebound.item.armor.ArmorTier;
+import com.mineboundteam.minebound.magic.PassiveSpellItem;
+import net.minecraft.ChatFormatting;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.chat.TextComponent;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.TooltipFlag;
+import net.minecraft.world.level.Level;
+import net.minecraftforge.common.ForgeConfigSpec.Builder;
+import net.minecraftforge.common.ForgeConfigSpec.IntValue;
+import net.minecraftforge.event.TickEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.event.config.ModConfigEvent;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.HashMap;
+import java.util.List;
+
+
+@Mod.EventBusSubscriber(modid = MineBound.MOD_ID)
+public class TelekineticUtilitySpell extends PassiveSpellItem {
+
+    private final int manaCost;
+
+    static HashMap<ArmorTier, Double> manaCostReductions = new HashMap<>() {{
+        put(ArmorTier.SUIT, 0.20);
+        put(ArmorTier.SYNERGY, 0.35);
+        put(ArmorTier.SINGULARITY, 0.5);
+    }};
+
+    public TelekineticUtilitySpell(Properties properties, TelekineticUtilitySpellConfig config) {
+        super(properties, config.LEVEL);
+
+        manaCost = config.MANA_COST.get();
+    }
+
+    @SubscribeEvent
+    public static void onPlayerTick(TickEvent.PlayerTickEvent event) {
+        if (event.side.isServer() && event.phase == TickEvent.Phase.START && !event.player.isCreative()) {
+            Player player = event.player;
+            List<TelekineticUtilitySpell> equippedTelekineticUtilitySpells = getEquippedSpellsOfType(TelekineticUtilitySpell.class, event.player);
```

Comment:
~~this is kinda long for a variable name, yes i know that's the java standard i'm just used to c names~~